sdk/js/generated/device-entity-map.json

```diff
@@ -415,7 +415,7 @@
   },
   "max_frequency": ["ns", "ns"],
   "min_frequency": ["ns", "ns"],
-  "multicast": ["ns", "read_only"],
```

Comment:
This file is generated from `sdk/js/generated/device-field-masks.json`. Changes to this file need to be sourced from there, otherwise this change will be lost, when `device-entity-map.json` is regenerated.

As far as I remember, `device-field-masks.json` is currently still generated by hand using via `pkg/ttnpb/field_mask_validation.go`